internal/app/go-filecoin/connectors/retrieval_market/client.go

```diff
@@ -127,6 +131,16 @@ func (r *RetrievalClientConnector) CreatePaymentVoucher(ctx context.Context, pay
 	return &v, nil
 }
 
+func (r *RetrievalClientConnector) WaitForPaymentChannelCreation(messageCID cid.Cid) (address.Address, error) {
+	// TODO https://github.com/filecoin-project/go-filecoin/issues/4034
+	panic("implement me")
```

Comment:
It's a little disturbing that these aren't tested.

Reply:
this is Coming Soon via my current branch